int16 voltage
int16 water
int16 current 